### ISSUE TITLE
BUG With Banned Players / With API Enabled?

### ISSUE THREAD

**Issue by ThousandX** [AscensionGameDev/Intersect-Engine#54]
**Describe the bug**
This bug caused the server to crash, perhaps it is a banned player trying to enter.
Or list player banned on API.
I am using a system to list the server ranking via api, which was made available on the AGD page

**Version Encountered**
Version 0.6.2.401

**To Reproduce**
I just activated the API system

**Expected behavior**

**Screenshots**

**Logs**
`2020-04-01 02:03:38.952 [Error] Received unhandled exception from Name:Intersect Server.exe
There are no context policies.
.
```
--------------------------------------------------------------------------------
2020-04-01 02:03:38.968 [Error] 
  Message: A second operation started on this context before a previous operation completed. Any instance members are not guaranteed to be thread safe.
    Stack:    at Microsoft.EntityFrameworkCore.Internal.ConcurrencyDetector.EnterCriticalSection()
   at Microsoft.EntityFrameworkCore.Query.Internal.LinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Intersect.Server.Database.PlayerData.Ban.Find(String ip) in C:\TeamCity\buildAgent\work\40dc5d27074313da\Intersect.Server\Database\PlayerData\Ban.cs:line 212
   at Intersect.Server.Database.PlayerData.Ban.CheckBan(User user, String ip) in C:\TeamCity\buildAgent\work\40dc5d27074313da\Intersect.Server\Database\PlayerData\Ban.cs:line 173
   at Intersect.Server.Database.PlayerData.Ban.CheckBan(String ip) in C:\TeamCity\buildAgent\work\40dc5d27074313da\Intersect.Server\Database\PlayerData\Ban.cs:line 187
   at Intersect.Server.Networking.Lidgren.ServerNetwork.HandleConnectionRequested(INetworkLayerInterface sender, IConnection connection) in C:\TeamCity\buildAgent\work\40dc5d27074313da\Intersect.Server\Networking\Lidgren\ServerNetwork.cs:line 88
   at Intersect.Network.LidgrenInterface.TryHandleInboundMessage() in C:\TeamCity\buildAgent\work\40dc5d27074313da\Intersect.Network\LidgrenInterface.cs:line 653
   at Intersect.Network.LidgrenInterface.TryGetInboundBuffer(IBuffer& buffer, IConnection& connection) in C:\TeamCity\buildAgent\work\40dc5d27074313da\Intersect.Network\LidgrenInterface.cs:line 228
   at Intersect.Network.AbstractNetwork.HandleInboundMessageAvailable(INetworkLayerInterface sender) in C:\TeamCity\buildAgent\work\40dc5d27074313da\Intersect (Core)\Network\AbstractNetwork.cs:line 198
   at Intersect.Network.LidgrenInterface.<.ctor>b__12_0(Object peer) in C:\TeamCity\buildAgent\work\40dc5d27074313da\Intersect.Network\LidgrenInterface.cs:line 140
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.QueueUserWorkItemCallback.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
```

--------------------------------------------------------------------------------
`

**Operating System:**
Windows Server 2016

**Additional context**

**Comment by lodicolo** [comment]
Should be fixed with PR #70 

If this happens again, reopen and attach new logs.

**Comment by ThousandX** [comment]
It keeps falling, but without giving errors in the logs, I found it strange.
Using the latest version # 70
